src/Sylius/Behat/Page/Admin/Country/CreatePageInterface.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Behat\Page\Admin\Country;
+
+use Behat\Mink\Exception\ElementNotFoundException;
+use Sylius\Behat\Page\Admin\Crud\CreatePageInterface as BaseCreatePageInterface;
+
+/**
+ * @author Arkadiusz Krakowiak <arkadiusz.krakowiak@lakion.com>
+ */
+interface CreatePageInterface extends BaseCreatePageInterface
+{
+    /**
+     * @param string $name
+     *
+     * @throws ElementNotFoundException
```

Comment:
Not really. Otherwise we will have two different conventions. We can open RFC for that right now.